core/src/main/java/org/eclipse/hono/util/TenantObject.java

```diff
@@ -207,6 +233,23 @@ private TrustAnchor getTrustAnchorForPublicKey(final JsonObject keyProps) {
         }
     }
 
+    /**
+     * Checks whether auto-provisioning is enabled for the given CA issuer DN.
```

Comment:
IMHO this is a little misleading as it is unclear which property will be checked. What about
> Checks whether auto-provisioning is enabled for a CA.
>
> @param subjectDn The subject DN of the CA to check.